test/test_local_db.js

```diff
@@ -261,36 +264,8 @@ var testReplicaIncorrectPassword = function (cb) {
 }
 
 var testparseMongoConnectionString = function (cb) {
```

Comment:
Yeah, I can do that. Tried to avoid making too many changes, but ultimately having named test cases is fairly standard so we should do it. Personally I like the Mocha BDD style, but won't go that far since I'm in a minority with eng it seems 😉 

So pretty...
```js
describe('#parseConnectionURL', () => {
  it('should fail if missing auth in RHMAP', fn)
  it('should pass even if missing auth locally', fn)
  it('should fail if no hosts are provided', fn)
  it('should fail if passed a garbage input string', fn)
})
```